sdk/tables/azure-data-tables/inc/azure/data/tables/internal/constants.hpp

```diff
@@ -0,0 +1,36 @@
+namespace Azure { namespace Data { namespace Tables { namespace _detail {
```

Comment:
License header missing

(also I don't believe we should be adding header files to to `/internal/` within `/inc/` for SDK packages)

Reply:
Also, I don't believe we should be adding header files to `/internal/` within `/inc/` for SDK packages. Internal is for other package dependencies within the repo (largely for core or common packages)

Consider keeping this in the `table_client.hpp` file.